### PR TITLE
Make risk encounter detail text show amount of days since last exposure

### PR DESCRIPTION
**Pull request by johannesrohwer**
This PR fixes a bug where the risk explanation text always showed two days since the last exposure (see JIRA 1815).

## Checklist

* [ ] This PR does not change text that hasn't been signed off with the RKI. Have a look at the pinned issue [440](https://github.com/corona-warn-app/cwa-app-ios/issues)
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed.
* [x] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+		<key>VARIABLE</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>one</key>
+			<string>En son %u gün önce, COVID-19 tanısı konan en az bir kişiyle daha uzun süreyle ve yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz daha yüksektir.</string>
+			<key>other</key>
+			<string>En son %u gün önce, COVID-19 tanısı konan en az bir kişiyle daha uzun süreyle ve yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz daha yüksektir.</string>
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict:18]: i do not speak turkish, we need someone to double check this.

Comment (haosap) [reply src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict:18]: Hi @oemerb ,
Could you please help us check it?
Thanks.

Comment (oemerb) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict:18]: Hi, 
it should be like this for one:
`En son bir gün önce, COVID-19 tanısı konan en az bir kişiye uzun süreyle yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz yüksektir.`
And for other:
`En son %u gün önce, COVID-19 tanısı konan en az bir kişiye uzun süreyle yakın mesafeden maruz kaldığınız için enfeksiyon riskiniz yüksektir.`

Comment (oemerb) [reply src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict:18]: Hi, 
added a change proposal below.

**Review by melloskitten — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict (hunk 1 above)
Comment (johannesrohwer) [reply src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.stringsdict]: thank you, I implemented your proposals!

**Review by inf2381 — APPROVED** [review]

**Pull request merged**